src/main/java/net/pms/util/OpenSubtitle.java

```diff
@@ -214,14 +218,19 @@ public static String computeHash(Path file) throws IOException {
 			return null;
 		}
 
+		String cachedHash = cacheUtil.getItem(file.getFileName().toString());
+		if(cachedHash!=null)return cachedHash;
```

Comment:
This format is in accordance with the CheckStyle.

    if (cachedHash != null) {
        return cachedHash;
    }